.github/workflows/push-helm.yaml

```diff
@@ -0,0 +1,51 @@
+name: Push Chart to OCI
+
+on:
+  push:
+    branches:
+      - coreweave
+
+jobs:
+  push-helm-chart:
+    runs-on: cw
```

Comment:
On CW runners a container needs to be specified for the job.  Like https://github.com/coreweave/github-actions-images?tab=readme-ov-file#github-helm-runner-image (which includes helm already).